client-side/src/components/date/formatDate.js

```diff
@@ -0,0 +1,10 @@
+import { format } from 'date-fns';
+
+const formatDate = (event,date) => {
+    const newFormat = event;
```

Comment:
please rename event to dateFormat, and remove the unnecessary newFormat variable

Reply:
you didn't change the variable event name